deployment/helm/ditto-mongodb-pvc.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: ditto-pvc
```

Comment:
@kaizimmerm Should we name the pvc also ditto-mongodb-pvc?

Reply:
fixed